### ISSUE TITLE
Show when a block is a legal order in results page

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/cmp-issues#46]
Where we know a block is the result of a copyright order, display the result in our results as "Blocked by court order"

**Comment by mkillock** [comment]
Is this knowledge represented in the results given by the API? @dantheta

**Comment by dantheta** [comment]
Not at the moment, I'm afraid. The probes know which rule matches a response page, but not necessarily what each rule means - that would have to be encoded in the config.  I can introduce that to the config file as a non-breaking change (at the expense of a little duplication and an increased version number).

If we're really keen on the feature, it's only 2-3 hours work to attach a "type" parameter for the match rule, carry it back with the response and return it in /status/url.  It's a little bit of paperwork for anyone else using the config file, but I think PyProbe is the main consumer for that at the moment.

@JimKillock, @graphiclunarkid  - are you keen for this feature to go in before launch?

**Comment by JimKillock** [comment]
It'd be a nice addition! But _only_ if you have time - it isn't essential.

**Comment by graphiclunarkid** [comment]
Closing in favour of #67, which has a better description of the feature.
